python/bind_data_access.h

```diff
@@ -70,17 +70,24 @@ class DataAccessHelper {
         return py::dtype::of<T>();
       }
     };
+    const auto get_base = [&view]() {
+      if constexpr (std::is_same_v<std::decay_t<View>, scipp::Variable>) {
+        return py::cast(view.data_handle());
+      } else {
+        return py::cast(view.data().data_handle());
+      }
+    };
```

Comment:
Same branching as above in `get_strides`, make a unified `get_data` helper instead?

Reply:
Not sure what you mean. The condition is the same as in `get_strides` but unless you want to make a function with two return values, capture them, and then pass them on to `py::array`, we cannot combine them.

Reply:
I didn't mean to combine the two functions, but to combine the branching:
```c++
    const auto get_data = (auto &&view) -> decltype(auto {
      if constexpr (std::is_same_v<std::decay_t<decltype(view)>, scipp::Variable>) {
        return view
      } else {
        return view.data()
      }
    };
```
then the two functions become either trivial/redundant, or simple one-liners.

Reply:
Gotcha!

Reply:
Done